client/blocks/jitm/templates/default.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+
+export default function DefaultTemplate( {
+	id,
+	CTA,
+	message,
+	description,
+	featureClass,
+	tracks,
+	trackImpression,
+	onClick,
+	onDismiss,
+} ) {
+	return (
+		<>
+			{ trackImpression && trackImpression() }
+			<Banner
+				callToAction={ CTA.message }
+				title={ message }
+				description={ description }
+				disableHref
+				dismissPreferenceName={ featureClass + '123' }
```

Comment:
Trailing `123` here sounds a bit random. Could we be more descriptive here? According to the readme https://github.com/Automattic/wp-calypso/blob/master/client/components/banner/README.md#L44, it will be automatically prefixed with `dismissible-card-`, so I think `featureClass` alone would be enough.

Reply:
The random number originally came from the previous implementation Jetpack team wrote. I just kept it as was. But, I totally agree with you. The `featureClass` would be enough.